lib/dialects/postgres/schema/pg-columncompiler.js

```diff
@@ -73,7 +73,7 @@ class ColumnCompiler_PG extends ColumnCompiler {
     }
 
     return `${useTz ? 'timestamptz' : 'timestamp'}${
-      precision ? '(' + precision + ')' : ''
+      precision || precision === 0 ? '(' + precision + ')' : ''
```

Comment:
according to PostgreSQL documentation, precision cannot be 0: https://www.postgresql.org/docs/10/datatype-numeric.html
What is the reason behind this change?